library/src/main/java/com/schibsted/spain/barista/interaction/BaristaListInteractions.kt

```diff
@@ -52,9 +53,15 @@ object BaristaListInteractions {
   @JvmStatic
   @JvmOverloads
   fun clickListItemChild(@IdRes id: Int? = null, position: Int, @IdRes childId: Int) {
+    performCustomActionOnListItemChild(id, position, childId, click())
+  }
+
+  @JvmStatic
+  @JvmOverloads
+  fun performCustomActionOnListItemChild(@IdRes id: Int? = null, position: Int, @IdRes childId: Int, viewAction: ViewAction) {
```

Comment:
Nice generalization! 
The name seems verbose though. I suggest a shorter version: 
```
doOnListItemChild(position = 1, childId = R.id.child, action = longClick()
```